### PR TITLE
Updated third-party apps sections

### PR DESCRIPTION
**Pull request by tamigoodall**
Updated links from other files
Consolidate files
Deleted 3 files
Added new guidance from SME

Review links:

- https://docs-content-staging-pr-8573.herokuapp.com/docs/applications
- https://docs-content-staging-pr-8573.herokuapp.com/docs/applications/concepts/app-types-first-third-party

### CODE REVIEW

**Comment by cocojoe** [comment]
Hey @tamigoodall what do you need to be reviewed from SDKs, I had a quick look but the content but most seems dashboard related. Thanks

**Comment by tamigoodall** [comment]
> Hey @tamigoodall what do you need to be reviewed from SDKs, I had a quick look but the content but most seems dashboard related. Thanks

Hi @cocojoe That review request was sent by mistake. Not sure how it was activated. Sorry ffor wasting your time.

File: articles/api-auth/apis.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The other available views for your API are:
 
 - **Scopes**: here you can define the scopes for this API, by setting a name and a description.
 
-- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications/concepts/app-types-auth0). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to  limit your authorized application's access. 
+- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to  limit your authorized application's access.
```
Comment (jeffreylees) [inline articles/api-auth/apis.md]:
```suggestion
- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled** for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit your authorized application's access. 
```

File: articles/api-auth/tutorials/client-credentials/customize-with-hooks.md [hunk 2]
```diff
@@ -15,15 +15,11 @@ useCase:
...
+# Use Hooks with Client Credentials Grant
 
 You can now add [Hooks](/hooks) into your [client credentials](/api-auth/grant/client-credentials) flow. This way you can change the <dfn data-key="scope">scopes</dfn> and add custom claims to the tokens issued by Auth0.
 
-## Overview
-
-Hooks allow you to customize the behavior of Auth0 using Node.js code.
-
-They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
+Hooks allow you to customize the behavior of Auth0 using Node.js code.They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
```
Comment (jeffreylees) [inline articles/api-auth/tutorials/client-credentials/customize-with-hooks.md]:
```suggestion
Hooks allow you to customize the behavior of Auth0 using Node.js code. They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
```

File: articles/applications/guides/enable-third-party-apps.md [hunk 3]
```diff
@@ -11,32 +10,37 @@ contentType:
...
+  
+You can enable third-party applications for your tenant. See [First-Party and Third-Party Applications](/applications/concepts/app-types-first-third-party) for details on the differences between the two types of applications. 
 
-This guide will show you how to enable [third-party applications](/applications/concepts/app-types-first-third-party#third-party-applications) for your tenant. 
+1. [Promote the connections you will use with third-party applications to domain level](/api/management/guides/connections/promote-connection-domain-level) in Auth0.
 
-## Steps 
+    Third-party applications can only authenticate users from [connections](/connections) flagged as domain-level connections. Domain-level connections can be enabled for selected first-party applications while also being open to all third-party application users for authentication.
+    
+2. Update your application's login page (if you use [Lock](/libraries/lock/v11) in the [Hosted Login Page](/hosted-pages/login), you must also:
```
Comment (jeffreylees) [inline articles/applications/guides/enable-third-party-apps.md]:
```suggestion
2. Update your application's login page. If you use [Lock](/libraries/lock/v11) in the [Universal Login Page](/universal-login/classic), you must also:
```

We don't say Hosted Pages anymore, and there was no close to this parenthetical

File: articles/applications/index.md [hunk 4]
```diff
@@ -1,41 +1,52 @@
...
 
-* Learn how to [remove an application using the Auth0 Dashboard](/dashboard/guides/applications/remove-app) or the [Management API](/api/management/guides/applications/remove-app).
+## Remove apps
 
-* Learn how to [rotate an application's Client Secret using the Auth0 Dashboard](/dashboard/guides/applications/rotate-client-secret) or the [Management API](/api/management/guides/applications/rotate-client-secret).
+You can [remove an application using the Auth0 Dashboard](/dashboard/guides/applications/remove-app) or the [Management API](/api/management/guides/applications/remove-app).
+
+## Manage client secrets
+
+You can [rotate an app's Client Secret](/dashboard/guides/applications/rotate-client-secret)  using the Auth0 Dashboard or the [Management API](/api/management/guides/applications/rotate-client-secret)
```
Comment (jeffreylees) [inline articles/applications/index.md]:
```suggestion
You can [rotate an app's Client Secret](/dashboard/guides/applications/rotate-client-secret) using the Auth0 Dashboard or the [Management API](/api/management/guides/applications/rotate-client-secret)
```

File: articles/applications/guides/enable-third-party-apps.md [hunk 5]
```diff
@@ -11,32 +10,37 @@ contentType:
...
 
-## Steps 
+    Third-party applications can only authenticate users from [connections](/connections) flagged as domain-level connections. Domain-level connections can be enabled for selected first-party applications while also being open to all third-party application users for authentication.
+    
+2. Update your application's login page. If you use [Lock](/libraries/lock/v11) in the [Universal Login Page](/universal-login/classic), you must also:
 
-To enable third-party applications for your tenant, you will need to:
+  - Upgrade to Lock version 11 or later
+  - Set the `__useTenantInfo: config.isThirdPartyClient` flag when instantiating Lock
+  - *For [Private Cloud](/private-cloud) users only*: Set the [`configurationBaseUrl` option](https://github.com/auth0/lock#other-options) to `https://{config.auth0Domain}/` when instantiating Lock
```
Comment (jeffreylees) [inline articles/applications/guides/enable-third-party-apps.md]:
```suggestion
  - *For [Private Cloud](/private-cloud) users only*: Set the [`configurationBaseUrl` option](https://auth0.com/docs/libraries/lock/v11/configuration#configurationbaseurl-string-) to `https://{config.auth0Domain}/` when instantiating Lock
```

File: articles/dashboard/guides/connections/view-connections.md [hunk 6]
```diff
@@ -29,5 +29,5 @@ This guide will show you how to view all of the available and configured [connec
 
 - [Set Up Social Connections](/dashboard/guides/connections/set-up-connections-social)
 - [Identity Providers Supported by Auth0](/identityproviders)
-- [Multiple Tenants](/applications/concepts/multiple-tenants)
+- [Create Multiple Tenants](/dashboard/guides/tenants/create-multiple-tenants.md)
```
Comment (jeffreylees) [inline articles/dashboard/guides/connections/view-connections.md]:
```suggestion
- [Create Multiple Tenants](/dashboard/guides/tenants/create-multiple-tenants)
```

File: articles/dashboard/reference/views-api.md [hunk 7]
```diff
@@ -20,6 +20,6 @@ Available views for your API include:
 
 - **Scopes**: allows you to [define the scopes](/scopes/current/guides/define-scopes-using-dashboard) for your API by setting scope names and descriptions. For more details, see [API Scopes](/scopes/current/api-scopes).
 
-- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications/concepts/app-types-auth0). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit an authorized application's access to your API. 
+- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit an authorized application's access to your API.
```
Comment (jeffreylees) [inline articles/dashboard/reference/views-api.md]:
```suggestion
- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled** for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit an authorized application's access to your API. 
```

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**